src/App.tsx

```diff
@@ -2,60 +1,129 @@
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todo } from './Interfaces/intTodo';
+import { TodoList } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import './App.scss';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [title, setTitle] = useState('');
+  const [isTitleError, setIsTitleError] = useState(false);
+  const [isUserError, setIsUserError] = useState(false);
+
+  const addNewTodo = (newTodo: Todo) => {
+    setTodos(prevTodos => [...prevTodos, newTodo]);
+  };
+
+  const onTitleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(e.target.value);
+    if (isTitleError && e.target.value.trim()) {
+      setIsTitleError(false);
+    } else if (!isTitleError && !e.target.value.trim()) {
+      setIsTitleError(true);
+    }
+  };
+
+  const onUserChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
Consider renaming the variable `e` to something more descriptive like `event` to improve code readability, as per the checklist recommendation: "Don't use one-letter variable naming."